visualization.py

```diff
@@ -0,0 +1,6 @@
+import matplotlib.pyplot as plt
+
+def monthly_spending_trend(df):
+    df[['Year','Month','Day']] = df['Date'].str.split('-', expand= True)
+    spending = df[df['Category'] != 'Income']
```

Comment:
```suggestion
    spending = df[df['Category'] == 'Expense']
```

I guess this is easier to understand.